mirascope/base/extractors.py

```diff
@@ -149,11 +180,8 @@ class TempCall(call_type):  # type: ignore
                 raise AttributeError("No tool found in the completion.")
             return extracted_schema
         except (AttributeError, ValueError, ValidationError) as e:
-            if retries > 0:
-                logging.info(f"Retrying due to exception: {e}")
-                # TODO: include failure in retry prompt.
-                return self._extract(call_type, tool_type, retries - 1, **kwargs)
-            raise  # re-raise if we have no retries left
+            logging.info(f"Retrying due to exception: {e}")
```

Comment:
if we use logfire do these logs get captured, or should we call `logfire.info` in that case?

Reply:
I think the user would try except the extract function and use log fire info and we can remove this line?

Reply:
This is logging each retry, which the user won't see, so they would only see their final log that there was an exception, not that there were X retries attempted in the logs.

If this shows up in logfire with no changes, great. Otherwise we should try to make it show up.

Reply:
This will not show up

Reply:
looks like logfire handler covers this now.